packages/core/content-type-builder/admin/src/translations/en.json

```
@@ -64,7 +64,7 @@
   "error.validation.enum-duplicate": "Duplicate values are not allowed (only alphanumeric characters are taken into account).",
   "error.validation.enum-empty-string": "Empty strings are not allowed",
   "error.validation.enum-regex": "At least one value is invalid. Values should have at least one alphabetical character preceding the first occurence of a number.",
-  "error.validation.minSupMax": "Can't be superior",
+  "error.validation.minSupMax": "min can't be superior to max",
```

```suggestion
  "error.validation.minSupMax": "Min can't be superior to max",
```

Should this still be title case?